GSoC-interest.md

```diff
@@ -23,5 +23,6 @@ GSoC Mentors
 | --- | --- | --- | --- | --- | --- |
 | Your Name Here | Your Email Here |  Idea You Hoping to Work On | Link to your Mico-task Repo | Link to Your Proposal |
 | Georg Link | linkgeorg@gmail.com | This is an example entry | [Micro-task](https://github.com/chaoss/governance/blob/master/GSoC-Ideas.md) | [Proposal Template](https://docs.google.com/document/d/1YZez6_hgp2dBybEsMZoQ-ONB9IawK4_OPISLHe9Tjew/edit) | NO |
+| Rashmi K A | k.a.rashmi04@gmail.com | Extend data model and user interface to capture better information about contributors | [Micro-tasks repository](https://github.com/Rashmi-K-A/chaoss-sortinghat) | [Proposal Template](https://docs.google.com/document/d/1oUd-G-N4VXh77FRI4PTSWJHoVxGt2WZDNu7JwuFMbW4/edit?usp=sharing) | NO |
```

Comment:
```suggestion
| Rashmi K A | k.a.rashmi04@gmail.com | Extend data model and user interface to capture better information about contributors | [Micro-tasks repository](https://github.com/Rashmi-K-A/chaoss-sortinghat) | [Proposal](https://docs.google.com/document/d/1oUd-G-N4VXh77FRI4PTSWJHoVxGt2WZDNu7JwuFMbW4/edit?usp=sharing) | NO |
```

Reply:
@GeorgLink I have shared my draft link on the GSoC dashboard. I haven't submitted the final proposal pdf since the mentors for the project haven't reviewed my proposal yet.  My understanding is that they will review and add comments/suggestions for the draft and I can submit the final pdf after that. Does this mean I can mark the last column as 'yes' now?